Jenkinsfile

```diff
@@ -437,20 +437,24 @@ pipeline {
                                                 ' --build-arg QUICKBUILD_DEPS="' +
                                                 quickBuildDeps('centos7') + '"' +
                                                 ' --build-arg REPOS="' + prRepos() + '"'
+                            args '--tmpfs /mnt/daos'
                         }
                     }
                     steps {
                         sconsBuild parallel_build: parallelBuild(),
                                    scons_exe: 'scons-3',
                                    scons_args: "PREFIX=/opt/daos TARGET_TYPE=release",
                                    build_deps: "no"
+                        sh (script:"""sudo ./utils/docker_nlt.sh --class-name centos7.release kv""",
```

Comment:
I'd like to change this from running the 'kv' test to the new tested added in https://github.com/daos-stack/daos/pull/5587 because kv is slightly higher risk because it can potentially deadlock on failure.  Both PRs have one review, I'll push for 5587 first them update this one for an easier landing.